charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -121,6 +121,15 @@ remote:
     basicAuth:
       username: ""
       password: ""
+
+    # -- Bearer token to use for authentication
+    bearer_token: ""
+
+    # -- Whether to verify TLS certificates when connecting to read endpoint
+    verify_tls: true
+
+    # -- List of extra filters to apply to all queries. See: https://docs.victoriametrics.com/#prometheus-querying-api-enhancements
+    extra_filters: [ ]
```

Comment:
what about extra labels? for both - reading and writing

Reply:
For reading `extra_filters` can already supersede `extra_label` parameter and we haven't seen a reason to add both atm as it wasn't requested.

For writing vmanomaly appends all fields passed in `metric_format`, e.g.:
```
metric_format:
  name: "node_exporter_$VAR"
  query_key: "for"
  model: prophet
```
Here `name` and `query_key` are reserved names, but `model` label will be passed as-is and will result in appending `model=prophet` to all emitted metrics.